src/connector/src/sink/catalog/desc.rs

```diff
@@ -140,4 +145,16 @@ impl SinkDesc {
             secret_refs: self.secret_refs.clone(),
         }
     }
+
+    pub fn is_fs_sink(&self) -> bool {
```

Comment:
To be more accurate, we should rename this to `is_file_sink` because `fs_sink` (filesystem sink) is only one possible type of file sink.